torchgeo/trainers/segmentation.py

```diff
@@ -93,7 +104,7 @@ def config_task(self) -> None:
                     _, state_dict = utils.extract_backbone(weights)
                 else:
                     state_dict = get_weight(weights).get_state_dict(progress=True)
-                self.model.encoder.load_state_dict(state_dict)
+                self.model.encoder.load_state_dict(state_dict, strict=False)
```

Comment:
This is needed because for the `tu-vit` model `self.model.encoder` has a `head.weight` and `bias.weight` which is not something that a pretrained model has.

Reply:
Additionally, the state dict of `self.model.encoder` is named `model.{module}` whereas resnet backbones for Unet are just `{model}`.